.github/workflows/dev-workflow.yml

```
@@ -225,6 +225,7 @@ jobs:
           MIGRATE: false
           ENABLE_S3: false
           ENABLE_LAMBDA: true
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

```suggestion
          SLACK_WEBHOOK_URL: ${{ secrets.DEV_SLACK }}
```
개발환경은 이 채널로 통일해주세요. 그리고 변수를 조금더 구체적으로 기술해주세요.